### PR TITLE
[MM-52237] - Update copy on compliance block screen

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Since we don't/won't have Cloud Free, compliance block screen should not mention "Cloud Free" anywhere

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-52237

#### Screenshots
Screenshot 2023-04-24 at 18 18 59

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/i18n/en.json [hunk 1]
```diff
@@ -315,7 +315,7 @@
   "admin.billing.subscription.cloudTrial.subscribeButton": "Upgrade Now",
   "admin.billing.subscription.cloudTrialBadge.daysLeftOnTrial": "{daysLeftOnTrial} trial days left",
   "admin.billing.subscription.cloudYearlyBadge": "Annual",
-  "admin.billing.subscription.complianceScreenFailed.button": "Continue with Cloud Free",
+  "admin.billing.subscription.complianceScreenFailed.button": "OK",
   "admin.billing.subscription.complianceScreenFailed.subtitle": "We will check things on our side and get back to you within 3 days once your Cloud subscription upgrade is approved. In the meantime, please feel free to continue using the free version of our product.",
```
Comment (ConorMacpherson) [inline webapp/channels/src/i18n/en.json:319]: Looks like the subtitle text can be removed as well

**Review by ConorMacpherson — APPROVED** [review]
One small change is needed. Otherwise, LGTM! :)

**Review by marianunez — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**